### ISSUE TITLE
0.5.12 breaks with subplots and horizontal colorbar

### ISSUE THREAD

**Issue by max3-2** [nschloe/matplotlib2tikz#115]
Since 0.5.12 update multiple subplots break with horizontal colorbar. Aspect ratio of bar getting inverted?

sample code python:
```python
import numpy as np
import matplotlib.pyplot as plt
import matplotlib2tikz

data = np.random.random((512,640))

fig = plt.figure()
ax1 = plt.subplot(121)
ax2 = plt.subplot(122)
ax = [ax1,ax2]

for i,axes in enumerate(ax):
    im = axes.imshow(data)
        
cax = fig.colorbar(im, ax=ax, orientation='horizontal')

matplotlib2tikz.save('2plots.tex')
```
produces tex:
```latex
% This file was created by matplotlib2tikz v0.5.12.
\begin{tikzpicture}

\begin{groupplot}[group style={group size=2 by 1}]
\nextgroupplot[
xmin=-0.5, xmax=639.5,
ymin=-0.5, ymax=511.5,
axis on top
]
\addplot graphics [includegraphics cmd=\pgfimage,xmin=-0.5, xmax=639.5, ymin=511.5, ymax=-0.5] {img3.png};
\begin{axis}[
xmin=-0.5, xmax=639.5,
ymin=-0.5, ymax=511.5,
axis on top,
colorbar horizontal,
colormap={mymap}{[1pt]
  rgb(0pt)=(0,0,0.5);
  rgb(22pt)=(0,0,1);
  rgb(25pt)=(0,0,1);
  rgb(68pt)=(0,0.86,1);
  rgb(70pt)=(0,0.9,0.967741935483871);
  rgb(75pt)=(0.0806451612903226,1,0.887096774193548);
  rgb(128pt)=(0.935483870967742,1,0.0322580645161291);
  rgb(130pt)=(0.967741935483871,0.962962962962963,0);
  rgb(132pt)=(1,0.925925925925926,0);
  rgb(178pt)=(1,0.0740740740740741,0);
  rgb(182pt)=(0.909090909090909,0,0);
  rgb(200pt)=(0.5,0,0)
},
point meta min=5.50205141325311e-06,
point meta max=0.99999961523477,
colorbar style={xtick={0.1,0.2,0.3,0.4,0.5,0.6,0.7,0.8,0.9},xticklabels={0.1,0.2,0.3,0.4,0.5,0.6,0.7,0.8,0.9}}
]
\addplot graphics [includegraphics cmd=\pgfimage,xmin=-0.5, xmax=639.5, ymin=511.5, ymax=-0.5] {img4.png};
\end{axis}

\end{tikzpicture}

```

with error:
"...nested axis not allowed..."

**Comment by nschloe** [comment]
Colorbars and subplots are hard; see http://stackoverflow.com/q/39719709/353337.

**Comment by nschloe** [comment]
Fix released (0.5.14).
